src/modules/client/components/planning/EventCreationModal.vue

```diff
@@ -191,6 +187,9 @@ export default {
     },
   },
   methods: {
+    update (event, path) {
+      this.$emit('update-event', { event, path });
+    },
```

Comment:
Que penses-tu de renommer cette fonction `updateEvent` et `event` en `value` ? Je trouve ça plus compréhensible comme ça : 
```
    updateEvent (path, value) {
      this.$emit('update-event', { event: value, path });
    },
```

Reply:
Tu as raison c'est peut être plus claire comme ca, par contre que dis-tu de plutot remplacer event par value? Je trouve que c'est plus clair de mettre a jour `set(this.newEvent, path, value)` que `set(this.newEvent, path, event)`, qu'en penses-tu?

Reply:
oui tout a fait !